install.lua

```diff
@@ -27,7 +27,10 @@ local function request(url_path)
 end
 
 local function makeFile(file_path, data)
-	local file = fs.open('github.rom/'..file_path,'w')
+	local path = 'github.rom/'..file_path
+	local dir = path:match('(.*/)')
+	fs.makeDir(dir)
```

Comment:
What does `makeDir` do if the directory already exists?

Reply:
Nothing. If you're talking about updating through running the script again, line 67/70 breaks it because the file already exists and won't be overwritten. I'll fix that in another PR.